apps/sms/services/test/unit/moz_mobile_message/moz_mobile_message_shim_test.js

```diff
@@ -50,39 +43,13 @@ suite('MozMobileMessageShim >', function() {
 
   test('bridge service is correctly initialized', function() {
     sinon.assert.calledOnce(bridge.service);
-    sinon.assert.calledWith(bridge.service, 'mozMobileMessageShim');
+    sinon.assert.calledWith(bridge.service, 'moz-mobile-message-shim');
     sinon.assert.calledWith(serviceStub.plugin, streamService);
-    sinon.assert.calledOnce(serviceStub.listen);
-  });
-
-  test('event listener is correctly initialized', function() {
-    EVENTS.forEach((event) => {
-      sinon.assert.calledWith(
-        MockNavigatormozMobileMessage.addEventListener,
-        event,
-        MozMobileMessageShim[Utils.camelCase(`on-${event}`)]
-      );
-    });
-  });
-
-  test('method mapping is correctly initialized', function() {
-    METHODS.forEach((shimMethod) => {
-      sinon.assert.calledWith(
-        serviceStub.method,
-        shimMethod,
-        MozMobileMessageShim[shimMethod]
-      );
-    });
-  });
-
-  test('stream mapping is correctly initialized', function() {
-    STREAMS.forEach((shimStream) => {
-      sinon.assert.calledWith(
-        serviceStub.stream,
-        shimStream,
-        MozMobileMessageShim[shimStream]
-      );
-    });
+    sinon.assert.calledTwice(serviceStub.listen);
+    sinon.assert.calledWith(
+      serviceStub.listen,
+      sinon.match.instanceOf(BroadcastChannel)
```

Comment:
__optional nit:__ same advice with ```BroadcastChannel``` name here as well, but I'll modify it in my patch anyway, so you can ignore :)